flexget/plugins/sites/torrentleech.py

```diff
@@ -23,12 +23,23 @@
     # Movies
     'Cam': 8,
     'TS': 9,
+    'TS/TC': 9,
+#    R5 is deprecated
     'R5': 10,
     'DVDRip': 11,
+	'DVDRip/DVDScreener': 11,
```

Comment:
What's with all the random indentations?

Reply:
Copy paste, Error, will fix